fdbclient/DatabaseConfiguration.h

```diff
@@ -32,16 +32,17 @@
 struct SatelliteInfo {
 	Key dcId;
 	int32_t priority;
+	int32_t satelliteDesiredTLogCount;
```

Comment:
```suggestion
	int32_t satelliteDesiredTLogCount = -1;
```
IMHO we should prefer initializing defaults outside of the constructor. Otherwise if someone adds a new constructor she or he will need to remember to initialize everything again (I would suggest to change this whenever you introduce a new member somewhere).